### PR TITLE
fix(docs): added missing package manager tabs in cli docs

### LINKED ISSUE #3605: docs: add more package runners
### Is your feature request related to a problem? Please describe.

Here https://nextui.org/docs/guide/cli#without-installation, there is a CLI script to install the package without global installation which uses `npx`.
What about the users using `pnpm`, `yarn`, or `bun`?

### Describe the solution you'd like

I would like to add different tabs for package runners like in this [page](https://nextui.org/docs/guide/cli#global-installation).

yarn - `yarn dlx`
pnpm - `pnpm dlx`
bun - `bun x`

### Describe alternatives you've considered

Add new tabs.

### Screenshots or Videos

Untitled

### PR DESCRIPTION
**Pull request by awesome-pro**
Closes #3605

## 📝 Description

> Add missing different package manager tabs

## ⛳️ Current behavior (updates)

Screenshot 2024-08-04 at 7 16 46 PM

Screenshot 2024-08-04 at 7 29 26 PM

## 🚀 New behavior

Screenshot 2024-08-04 at 7 14 32 PM

Screenshot 2024-08-04 at 7 17 19 PM

## Summary by CodeRabbit

- **New Features**
	- Introduced a new `<PackageManagers>` component for improved presentation of package manager commands in the documentation.
	- Updated installation and server start commands to utilize the `<PackageManagers>` component, enhancing clarity and consistency.

- **Documentation**
	- Replaced inline code blocks with the `<PackageManagers>` component in documentation for better readability and maintainability.

### CODE REVIEW

**Review by wingkwong — CHANGES_REQUESTED** [review]
L33 needs to be updated as well.

**Review by wingkwong — APPROVED** [review]

File: apps/docs/content/docs/guide/cli.mdx [hunk 1]
```diff
@@ -30,11 +30,16 @@ To install the `CLI` globally, execute one of the following commands in your ter
...
 
-```bash
-npx nextui-cli@latest
-```
+<PackageManagers
+  commands={{
+    npm: "npx nextui-cli@latest",
+    yarn: "yarn dlx nextui-cli@latest",
+    pnpm: "pnpm dlx nextui-cli@latest",
+    bun: "bunx nextui-cli@latest",
```
Comment (Jay-Karia) [inline apps/docs/content/docs/guide/cli.mdx:40]: `bunx` does not work. Instead we need to use: `bun x`

Comment (Jay-Karia) [reply apps/docs/content/docs/guide/cli.mdx:40]: Maybe it does not work on my end 🙃

bunx

`bun x` always works

bun x

Comment (wingkwong) [reply apps/docs/content/docs/guide/cli.mdx:40]: seems a known issue for bun on Windows. I saw there is a issue reported in bun repo.

**Review by wingkwong — CHANGES_REQUESTED** [review]
why reverting all changes?

**Comment by awesome-pro** [comment]
I was due to a small git  branch mistake,
I will resolve soon

**Pull request closed without merging**